native/cocos/2d/renderer/Batcher2d.cpp

```diff
@@ -44,30 +53,12 @@ Batcher2d::Batcher2d(Root* root)
     _root = root;
     _device = _root->getDevice();
     _stencilManager = StencilManager::getInstance();
+    instance = this;
```

Comment:
Seems not correct. should use a static function to do this. Maybe add a static initialize function to create batcher and set it to instance.